tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -180,8 +179,6 @@ public class ExampleGenerator {
 
         calendar.set(1999, 8, 5, 7, 0, 0); // 1999-09-05 07:00:00
```

Comment:
Hm. These actually will vary by locale, see:

- https://www.unicode.org/cldr/charts/latest/supplemental/day_periods.html 
- https://github.com/unicode-org/cldr/blob/main/common/supplemental/dayPeriods.xml
- from the class `SupplementalDataInfo` there is `getDayPeriodLocales()` and `getDayPeriods()` that should give you the period information.

That said, I think this is an improvement and we could go forward with it- but perhaps a followon to try to generate the dates based on the rules.